script/e2e-test.sh

```diff
@@ -165,38 +165,18 @@ installOrUpgradeKubeapps() {
     # Install Kubeapps
     info "Installing Kubeapps..."
     kubectl -n kubeapps delete secret localhost-tls || true
-    
+
     helm upgrade --install kubeapps-ci --namespace kubeapps "${chartSource}" \
       ${invalidateCacheFlag} \
       "${img_flags[@]}" \
       "${@:2}" \
+      "${multiclusterFlags[@]+"${multiclusterFlags[@]}"}" \
```

Comment:
why is not this line just?:

```
      "${multiclusterFlags[@]}" \
```

Reply:
See https://stackoverflow.com/a/61551944/4178335 and #2651